### PR TITLE
Marketing site update

### PR DESCRIPTION
**Pull request by forgetso**

### CODE REVIEW

File: src/_includes/navbar.njk [hunk 1]
```diff
@@ -1,88 +1,46 @@
...
-                <li class="nav-item">
-                    <a class="nav-link page-scroll" href="{{ site.url }}/news">Blog </a>
-                </li>
-                <li class="nav-item">
-                    <a class="nav-link page-scroll" href="#subscribe-form">Signup</a>
-                </li>
-            </ul>
 
+<div class="navbar w-full mx-auto bg-white border-b 2xl:max-w-7xl z-20">
+    <div x-data="{ open: false }" class="relative flex flex-col w-full p-5 mx-auto bg-white md:items-center md:justify-between md:flex-row md:px-6 lg:px-8">
```
Comment (forgetso) [inline src/_includes/navbar.njk:4]: Are these tags meant to make the navbar work with css?
```
x-data="{ open: false }"
```
```
@click="open = !open"
```

**Review by forgetso — COMMENTED** [review]
Just need some clarification on extra attributes.

File: src/assets/scripts/formValidation.js [hunk 2]
```diff
@@ -28,9 +28,9 @@ const validateUrl = function (url) {
 
 // Check the form elements are all filled before enabling the submit button
 window.checkForm = function () {
-    const form = document.getElementById('subscribe-form')
-    const inputs = Array.from(form.getElementsByTagName('input'))
-    const submit = document.getElementById('subscribe-submit')
+    const form = document.getElementById('signup')
+    const inputs = Array.from(form.getElementsByTagName('INPUT'))
```
Comment (goastler) [inline src/assets/scripts/formValidation.js:32]: why all caps?

**Review by goastler — CHANGES_REQUESTED** [review]

File: src/assets/scripts/formValidation.js (hunk 2 above)
Comment (forgetso) [reply src/assets/scripts/formValidation.js:32]: Was just debugging. If you look at the names returned by JS, it actually stores them in caps.

```
document.body.nodeName
'BODY'
```

Comment (goastler) [reply src/assets/scripts/formValidation.js:32]: oh, fair enough :)

**Review by goastler — APPROVED** [review]

**Pull request merged**